open-company/open-company-web
# Date separators

Card: https://trello.com/c/KlbIHxAh

To test:
- with a clean org
- open Recent
- from REPL change the first post date to be published (and created) last week
- refresh Recent
- do you NOT see a separator?
- add a new post
- do you see a separator now?
- does it have the right height? and the posts?
- generate some posts with the support script
- add a few posts to populate today too
- check the posts publish date to make sure they are in the right group
